troi/tools/spotify_lookup.py

```diff
@@ -134,13 +134,24 @@ def submit_to_spotify(spotify, playlist, spotify_user_id: str, is_public: bool =
 
     logger.info("submit %d tracks" % len(spotify_track_ids))
 
+    # Truncate to character limit for title
+    playlist_name = playlist.name[0:100]
```

Comment:
This may look kinda weird. Maybe cut to 97 and then add "..." ? Fine to merge now, we can improve this later.

Reply:
Yes I'll opt for merging for now, I think 100+ chars is an edge case I'll be happy to fix if someone complains.